lib/components/UploadPicker.vue

```diff
@@ -444,28 +460,33 @@ export default defineComponent({
 
 		updateStatus() {
 			if (this.isPaused) {
-				this.timeLeft = t('paused')
+				this.status = t('paused')
+				return
+			}
+
+			if (this.isOnlyAssembling) {
+				this.status = t('assembling')
 				return
 			}
 
-			const estimate = Math.round(this.eta!.estimate())
+			const estimate = Math.round(this.eta?.estimate?.() || 0)
 
 			if (estimate === Infinity) {
-				this.timeLeft = t('estimating time left')
+				this.status = t('estimating time left')
 				return
 			}
 			if (estimate < 10) {
-				this.timeLeft = t('a few seconds left')
+				this.status = t('a few seconds left')
 				return
 			}
 			if (estimate > 60) {
 				const date = new Date(0)
 				date.setSeconds(estimate)
 				const time = date.toISOString().slice(11, 11 + 8)
-				this.timeLeft = t('{time} left', { time }) // TRANSLATORS time has the format 00:00:00
+				this.status = t('{time} left', { time }) // TRANSLATORS time has the format 00:00:00
 				return
 			}
-			this.timeLeft = t('{seconds} seconds left', { seconds: estimate })
+			this.status = t('{seconds} seconds left', { seconds: estimate })
```

Comment:
Should this not be like this?

```suggestion
			this.status = n('{seconds} second left', '{seconds} seconds left', estimate, { seconds: estimate })
```